dietpi/dietpi-software

```diff
@@ -6976,7 +7002,18 @@ exec sudo -u $ha_user dash -c '$ha_pyenv_activation; exec pip3 install -U homeas
 			G_EXEC curl -sSfL "$INSTALL_URL_ADDRESS" -o get-pip.py
 			G_EXEC_OUTPUT=1 G_EXEC python3 ./get-pip.py
 			G_EXEC_NOHALT=1 G_EXEC rm get-pip.py
+			
+			# create piwheels config file for ARMv6 and ARMv7
+			if (( $G_HW_ARCH == 1 || $G_HW_ARCH == 2 )); then
+			 
+				if ! [[ -f '/etc/pip.conf' ]]; then				
+				
+					echo '[global]
+extra-index-url=https://www.piwheels.org/simple
+' > /etc/pip.conf
```

Comment:
Let's do that step prior to `get-pip.py`, as, if I'm not wrong, `get-pip.py` makes use of pre-compiled wheels as well. At least it does not hurt.

Reply:
ok changes, looks good

```
[  OK  ] DietPi-Software | curl -sSfL https://bootstrap.pypa.io/get-pip.py -o get-pip.py
[ INFO ] DietPi-Software | python3 ./get-pip.py, please wait...
Looking in indexes: https://pypi.org/simple, https://www.piwheels.org/simple
Collecting pip
  Downloading https://www.piwheels.org/simple/pip/pip-21.0-py3-none-any.whl (1.5 MB)
Collecting setuptools
  Downloading https://www.piwheels.org/simple/setuptools/setuptools-52.0.0-py3-none-any.whl (784 kB)
Collecting wheel
  Downloading https://www.piwheels.org/simple/wheel/wheel-0.36.2-py2.py3-none-any.whl (35 kB)
Installing collected packages: wheel, setuptools, pip
Successfully installed pip-21.0 setuptools-52.0.0 wheel-0.36.2
[  OK  ] DietPi-Software | python3 ./get-pip.py
[  OK  ] DietPi-Software | rm get-pip.py
```